.gitmessage

```diff
@@ -0,0 +1,13 @@
+# If applied, this commit will...
+
+Why is this change needed?
+--------------------------
+
+How does it address the issue?
+------------------------------
+
+Any links to any relevant tickets, articles or other resources?
+---------------------------------------------------------------
+
+# Was this co-authored?  Uncomment the following line, and make sure you use their official github email address
+# Co-authored-by: Tony Dewan tony.dewan@gmail.com
```

Comment:
I think you missed something important in the co-authoriship line. Specifically, I think you [need](https://help.github.com/articles/creating-a-commit-with-multiple-authors/) `<>` around the email, e.g:

```
Co-authored-by: Nathan Acuff <nathan@expectedbehavior.com>
```

Reply:
Also, it might be wise to have the comment block include all of our emails, so you just uncomment the one you need.  If I had guessed tony's off the top of my head, I would have been wrong.

Reply:
I updated this to be easier to work with and have the correct email address format.

Reply:
(in  https://github.com/expectedbehavior/common-files/commit/d3a87b4a058aec0f098cd0844ce86085829faac6)